varnish-mixin/dashboards/varnish-overview.libsonnet

```diff
@@ -0,0 +1,1305 @@
+  description: 'Backend logs for Varnish Cache.',
+  options: {
+    dedupStrategy: 'none',
+    enableLogDetails: true,
+    prettifyLogMessage: false,
+    showCommonLabels: false,
+    showLabels: false,
+    showTime: false,
+    sortOrder: 'Descending',
+    wrapLogMessage: false,
+  },
+};
+
+{
+  grafanaDashboards+:: {
+    'varnish-overview.json':
+      dashboard.new(
+        'Varnish overview',
+        time_from='%s' % $._config.dashboardPeriod,
+        tags=($._config.dashboardTags),
+        timezone='%s' % $._config.dashboardTimezone,
+        refresh='%s' % $._config.dashboardRefresh,
+        description='',
+        uid=dashboardUid,
+      )
+
+      .addTemplates(
+        std.flattenArrays([
+          [
+            template.datasource(
+              promDatasourceName,
+              'prometheus',
+              null,
+              label='Data Source',
+              refresh='load'
+            ),
+          ],
+          if $._config.enableLokiLogs then [
+            template.datasource(
+              lokiDatasourceName,
+              'loki',
+              null,
+              label='Loki Datasource',
+              refresh='load'
+            ),
+          ] else [],
+          [
+            template.new(
+              'job',
+              promDatasource,
+              'label_values(varnish_main_sessions,job)',
+              label='Job',
+              refresh=2,
+              includeAll=true,
+              multi=true,
+              allValues='',
+              sort=0
+            ),
+            template.new(
+              'instance',
+              promDatasource,
+              'label_values(varnish_main_sessions,instance)',
+              label='Instance',
+              refresh=2,
+              includeAll=true,
+              multi=true,
+              allValues='',
+              sort=0
+            ),
+          ],
+        ])
+      )
+      .addPanels(
+        std.flattenArrays([
+          [
+            cacheHitRatePanel { gridPos: { h: 4, w: 3, x: 0, y: 0 } },
+            frontendRequestsPanel { gridPos: { h: 4, w: 3, x: 3, y: 0 } },
+            backendRequestsPanel { gridPos: { h: 4, w: 3, x: 6, y: 0 } },
+            sessionsPanel { gridPos: { h: 4, w: 3, x: 9, y: 0 } },
+            cacheHitsPanel { gridPos: { h: 4, w: 3, x: 12, y: 0 } },
+            cacheHitPassPanel { gridPos: { h: 4, w: 3, x: 15, y: 0 } },
+            sessionQueueLengthPanel { gridPos: { h: 4, w: 3, x: 18, y: 0 } },
+            poolsPanel { gridPos: { h: 4, w: 3, x: 21, y: 0 } },
+            backendConnectionsPanel { gridPos: { h: 8, w: 12, x: 0, y: 4 } },
+            sessionsPanel { gridPos: { h: 8, w: 12, x: 12, y: 4 } },
+            memoryUsedPanel { gridPos: { h: 7, w: 12, x: 0, y: 12 } },
+            cachePanel { gridPos: { h: 7, w: 12, x: 12, y: 12 } },
+            requestsPanel { gridPos: { h: 8, w: 12, x: 0, y: 19 } },
+            cacheHitRatioPanel { gridPos: { h: 8, w: 12, x: 12, y: 19 } },
```

Comment:
Small nitpick but can you please swap the positions of the `cachePanel` and the `cacheHitRatio` panel? Just to reflect importance

Reply:
Yea sounds good. I think i would also have to move the requests panel up with cache hit ratio panel because those two correlate and the memory used correlates heavily with cache panel. Do you agree?

Reply:
it would look like this
<img width="3277" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/38274348/9af961bd-5aba-4887-afe6-9cc66ae3b26d">

Reply:
Yep that looks great! :)